### PR TITLE
Relay example: Add CI configuration to run unit tests & flow

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
Related to this MR: https://github.com/adeira/relay-example/pull/6

### CODE REVIEW

**Comment by mrtnzlml** [comment]
The point of the mentioned PR was also to fix the Importit script. Did you check it as well or is it a copy-paste?

**Comment by jaroslav-kubicek** [comment]
> The point of the mentioned PR was also to fix the Importit script. Did you check it as well or is it a copy-paste?

it's copypaste, but now I also tried running importit script and it seemed to work, is anything more expected?

**Comment by mrtnzlml** [comment]
> is anything more expected?

Yeah, you should be importing it via the script 😛

File: src/monorepo-shipit/config/example-relay.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ module.exports = {
       ['src/example-relay/__github__/babel.config.js', 'babel.config.js'],
       ['src/example-relay/__github__/jest.config.js', 'jest.config.js'],
       ['src/example-relay/__github__/flow-typed', 'flow-typed'],
+      ['src/example-relay/__github__/.github', '.github'],
```
Comment (mrtnzlml) [inline src/monorepo-shipit/config/example-relay.js]: Tests would be great. See: https://github.com/adeira/universe/pull/231/files

Comment (jaroslav-kubicek) [reply src/monorepo-shipit/config/example-relay.js]: fixed!

**Pull request merged**